src/quo2/components/selectors/styles.cljs

```diff
@@ -1,130 +1,175 @@
-     (colors/alpha colors/white (if disabled? 0.05 0.1)))
-    :else
-    (colors/theme-colors
-     (colors/alpha colors/neutral-20 (if disabled? 0.4 1))
-     (colors/alpha colors/neutral-70 (if disabled? 0.3 1)))))
+(defn toggle-background-color
+  [custom-color]
+  (let [on             (colors/custom-color-by-theme custom-color 50 60)
+        off            (colors/theme-colors colors/neutral-30 colors/neutral-80)
+        on-blur-light  (colors/custom-color custom-color 50)
+        on-blur-dark   colors/white-opa-70
+        off-blur-light colors/neutral-80-opa-20
+        off-blur-dark  colors/white-opa-10]
+    {:normal {:enabled  {:on  on
+                         :off off}
+              :disabled {:on  (colors/alpha on 0.3)
+                         :off (colors/alpha off 0.3)}}
+     :blur   {:enabled  {:on  (colors/theme-colors on-blur-light on-blur-dark)
+                         :off (colors/theme-colors off-blur-light off-blur-dark)}
+              :disabled {:on  (colors/theme-colors (colors/alpha on-blur-light 0.3)
+                                                   (colors/alpha on-blur-dark 0.21))
+                         :off (colors/theme-colors (colors/alpha off-blur-light 0.06)
+                                                   (colors/alpha off-blur-dark 0.03))}}}))
 
-(defn checkbox-prefill
-  [blurred-background? disabled?]
-  {:height           21
-   :width            21
-   :border-radius    6
-   :background-color (if blurred-background?
-                       (colors/theme-colors
-                        (colors/alpha colors/neutral-80
-                                      (if disabled? 0.05 0.1))
-                        (colors/alpha colors/white (if disabled? 0.05 0.1)))
-                       (colors/theme-colors
-                        (colors/alpha colors/neutral-20
-                                      (if disabled? 0.3 1))
-                        (colors/alpha colors/neutral-70
-                                      (if disabled? 0.3 1))))})
+(defn radio-border-color
+  [custom-color]
+  (let [on             (colors/custom-color-by-theme custom-color 50 60)
+        off            (colors/theme-colors colors/neutral-30 colors/neutral-70)
+        on-blur        (colors/theme-colors (colors/custom-color custom-color 50) colors/white)
+        off-blur-light colors/neutral-80-opa-20
+        off-blur-dark  colors/white-opa-40]
+    {:normal {:enabled  {:on  on
+                         :off off}
+              :disabled {:on  (colors/alpha on 0.3)
+                         :off (colors/alpha off 0.3)}}
+     :blur   {:enabled  {:on  on-blur
+                         :off (colors/theme-colors off-blur-light off-blur-dark)}
+              :disabled {:on  (colors/alpha on-blur 0.3)
+                         :off (colors/theme-colors (colors/alpha off-blur-light 0.06)
+                                                   (colors/alpha off-blur-dark 0.12))}}}))
```

Comment:
It's a convention to add a question mark at the end of booleans
`on? off? enabled? disabled?` and so on ..